2020/2020_General_Election_Court_of_Appeals_results_by_precinct/SHAWNEE.csv

```diff
@@ -0,0 +1,219 @@
+PRECINCT NAME,"Court of Appeals, Position No. 4","Court of Appeals, Position No. 4","Court of Appeals, Position No. 6","Court of Appeals, Position No. 6","Court of Appeals, Position No. 8","Court of Appeals, Position No. 8","Court of Appeals, Position No. 9","Court of Appeals, Position No. 9","Court of Appeals, Position No. 14","Court of Appeals, Position No. 14"
+,Yes,No,Yes,No,Yes,No,Yes,No,Yes,No
```

Comment:
rather than combine these yes/no header lines, I left them as is and will resolve them when converting in data-ks repo.